HBN/README.md

```diff
@@ -29,10 +29,12 @@ Note: Some of the following COINS access instructions were copied from [the HBN
     ```shell
     HBN/
     ├── all_data_dicts_Aug_2018.zip
-    ├── Data Dictionaries/
+    ├── data_dictionaries/
```

Comment:
@JessyD This should be `Data Dictonaries/` as that is what it is called when you unzip it as-is from the HBN site.